mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -30,27 +31,34 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	FIELD f_eyfbjeig horizontalSpeed F
 	FIELD f_eyihyrff pistonMovementDelta [D
 	FIELD f_fbrwyrto inNetherPortal Z
+	FIELD f_fhrqlvrj VELOCITY_AFFECTING_POS_Y_OFFSET D
 	FIELD f_fngqrxks standingEyeHeight F
 	FIELD f_gpwlbvdc lastNetherPortalPosition Lnet/minecraft/unmapped/C_hynzadkk;
+	FIELD f_gzzxrhqg BOARDING_COOLDOWN I
```

Comment:
Boarding? "You are now boarding delta airlines" type beat

Reply:
Not entirely sure what this means, I cant tell if it means mounting or something else because moj in general has mounting right so

Reply:
Yarn has MAX_RIDING_COOLDOWN...
RIDING_COOLDOWN? MOUNTING_COOLDOWN?

Reply:
`RIDING_COOLDOWN`

Reply:
What oro said